sites/wonderland/docs/development/solidity/onboarding/getting-started.md

```diff
@@ -8,7 +8,7 @@ This onboarding process is designed to provide you with the necessary knowledge,
 
 The onboarding is divided into two key sections:
 
-- [Knowledge Base](/docs/development/solidity/onboarding/knowledge-base): A comprehensive guide covering coding standards, best practices, and essential tools.
```

Comment:
I think it would be better to keep the `knowledge base` section as 1 page like it was and add a new section under `Coding Standards` that will have a small description and link to a new page to the side bar on the solidity sections like the "Coding Style" and "Natspec" one.
I see that the UI onboarding is also using the Defi and Dapps section, so doing what I describe above would benefit them also with just linking to the new page we create https://handbook.wonderland.xyz/docs/development/frontend/onboarding/front-knowledge-base#defi--dapps